docs/lib/auth/fragments/android/password_management/20_confirm_reset_password.md

```diff
@@ -14,12 +14,12 @@ Amplify.Auth.confirmResetPassword(
  <amplify-block name="Kotlin">
 
 ```kotlin
-Amplify.Auth.confirmResetPassword(
-   "NewPassword123",
-   "confirmation code you received",
-   { Log.i("AuthQuickstart", "New password confirmed") },
-   { error -> Log.e("AuthQuickstart", error.toString()) }
-)
+try {
```

Comment:
Is there a reason to have all the try/catch blocks in here?  If this is code for examples why not just pass or ignore exception handling unless it's central to the theme?

Reply:
Thanks Ken! I thought about this periodically over the weekend. I think there are pros and cons here.

As you note, there's a lot of extra boilerplate for error handling, when it really _isn't_ the main theme of this documentation.

On the other hand, these snippets each live in a widget that can switch languages. The choices are Java, RxJava, and Kotlin. The Java example, at least, _has to_ show error handling since it has an explicit error callbacks. For this reason, it might be best for all three to demonstrate error handling.

I suppose I'll leave the `try`/`catch` now, but may re-evaluate this soon.